src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import React from 'react';
+import { TabsProps } from './interface';
+
+const Tabs: React.FC<TabsProps> = ({ tabs, isActive, setIsActive }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <h1 className="title">
+        {`Selected tab is ${tabs[isActive].title}`}
+      </h1>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tabItem, index) => {
+            return (
+              <li
+                className={`${index === isActive ? 'is-active' : ''}`}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally